Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino

```diff
@@ -326,7 +310,7 @@ void doPublishOdom() {
   double vth = (vr - vl)/(2 * TRACK_RADIUS);
   
   // Calculate the deltas since
-  double dt = current_time.toSec() - last_odom_publish_time.toSec();
+  double dt = current_time.toSec() - last_raw_odom_publish_time.toSec();
```

Comment:
It is only used in this function, but it is initialized in setUp().  We use the difference between the current time and the last time to calculate deltas. Is there a way to declare it static in this function but initialize it?